Library/Homebrew/env_config.rb

```diff
@@ -318,6 +318,10 @@ module EnvConfig
                      "outdated.",
         boolean:     true,
       },
+      HOMEBREW_PIP_INDEX_URL:                    {
+        description:  "If set, `brew install <formula>` will use this URL to download PyPI package resources.",
+        default_text: "`https://pypi.org/simple`.",
```

Comment:
This would be unset by default. `https://pypi.org/packages` and `https://files.pythonhosted.org/packages` (this is what we use in formulae) are mutually identical. We use `https://pypi.org/simple` (suffix as `simple` rather than `package`) in the docs here because `https://files.pythonhosted.org/simple` is private.